src/plutus_gen/extraction/utils.rs

```diff
@@ -42,7 +42,7 @@ where
     }
 }
 
-fn convert_polynomial<W: std::io::Write>(
+fn convert_to_plutus_polynomial<W: std::io::Write>(
```

Comment:
name it properly please. Plutus is a low-level language to which both Aiken and Plinth are compiled.
I guess what you really do here is converting to Plinth